http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -860,6 +860,14 @@ paths:
           required: false
           schema:
             type: string
+        - in: query
+          name: creator
```

Comment:
key should `role`?

Reply:
This is to mirror your changes to the listRuntimes endpoints - role=creator and creator=me@my.email are both supported

Reply:
ahhh ok! will do